Source/DafnyLanguageServer.Test/Various/CounterExampleTest.cs

```diff
@@ -58,14 +78,17 @@ method Abs(x: int) returns (y: int)
       await SetUp(o => o.Set(CommonOptionBag.RelaxDefiniteAssignment, true));
       var documentItem = CreateTestDocument(source);
       await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
-      var counterExamples = (await RequestCounterExamples(documentItem.Uri)).ToArray();
+      var counterExamples = (await RequestCounterExamples(documentItem.Uri)).
+        OrderBy(counterexample => counterexample.Position).ToArray();
```

Comment:
Wonderful. This has been a consistent source of issues.

Reply:
I didn't realize that was the issue until one of the tests was failing with the new encoding - it is a good variability stress test.